### PR TITLE
removes explicit font definition for link component

### PR DESCRIPTION
**Pull request by AlanBreck**

### CODE REVIEW

File: src/components/link/link.svelte [hunk 1]
```diff
@@ -40,7 +40,7 @@
     --focus-shadow: var(--leo-link-focus-shadow, var(--leo-effect-focus-state));
 
     color: var(--color);
-    font: var(--leo-font-primary-default-semibold);
+    font: var(--leo-link-font, inherit);
```
Comment (fallaciousreasoning) [inline src/components/link/link.svelte:43]: We can probably just completely remove this

**Review by fallaciousreasoning — APPROVED** [review]
LGTM!

**Review by petemill — APPROVED** [review]

File: src/components/link/link.stories.svelte [hunk 2]
```diff
@@ -28,6 +28,11 @@
       type: 'string',
       defaultValue: 'Download Brave'
     },
+    '--leo-link-font': {
+      type: 'string',
+      control: 'text',
+      description: 'The font of the link'
+    },
```
Comment (AlanBreck) [inline src/components/link/link.stories.svelte:35]:
```suggestion
```

File: src/components/link/link.svelte (hunk 1 above)
Comment (AlanBreck) [reply src/components/link/link.svelte:43]:
```suggestion
```

**Review by fallaciousreasoning — APPROVED** [review]
Still LGTM!

**Pull request merged**